cmd/snap-update-ns/change.go

```diff
@@ -178,17 +180,28 @@ func (c *Change) ensureSource(sec *Secure) error {
 			}
 		}
 	} else if os.IsNotExist(err) {
-		_, err = c.createPath(path, false, sec)
+		// NOTE: This createPath is _not_ using pokeHoles so it will _not_
```

Comment:
This comment, while nice and descriptive, is a bit confusing on first read since you emphasize 'not' and then talk about pokeHoles. It might be easier to say that you'll use pokeholes only for layouts (and why) and then say emphasize 'not' for everything else. Not a blocker, but would be nice to clear up.